bindings/matrix-sdk-ffi/src/room.rs

```diff
@@ -320,15 +313,6 @@ impl Room {
         fully_read_event_id: String,
         read_receipt_event_id: Option<String>,
     ) -> Result<(), ClientError> {
-        let room = match &self.inner {
-            SdkRoom::Joined(j) => j.clone(),
-            _ => {
-                return Err(
-                    anyhow!("Can't send read markers to room that isn't in joined state").into()
```

Comment:
Can we have another check based on the room's state, for that? Any way to prevent confusing behavior, even at runtime, seems worth keeping IMO.

Reply:
If we did add that, it shouldn't be in the bindings. But I'd rather just not do it at all. By duplicating validations the server does anyways, we risk reporting an error when an action could actually have succeeded.